packages/form-js-viewer/src/render/components/form-fields/Fieldset.js

```diff
@@ -0,0 +1,41 @@
+import { useContext } from 'preact/hooks';
```

Comment:
A thing which probably doesn't come from these changes but from previous investigations of nested layout:

The parent is also highlighted when hovering over children. I'd assume only the children are highlighted.

![Kapture 2022-05-23 at 13 22 17](https://user-images.githubusercontent.com/9433996/169808675-f21235a9-f8a8-4b24-a709-e8a2c04d2e43.gif)